File: src/java/foundgine-core/src/main/java/com/foundgine/core/execution/ExecutionContextKeys.java
package com.foundgine.core.execution;

/**
 * Port of {@code Foundgine.Core.Execution.ExecutionContextKeys}: reserved
 * runtime context keys used by provider execution for dynamic request values.
 */
public final class ExecutionContextKeys {

	public static final String PAGINATION_LIMIT = "foundgine.pagination.limit";
	public static final String PAGINATION_OFFSET = "foundgine.pagination.offset";
	public static final String PAGINATION_HAS_CURSOR = "foundgine.pagination.hasCursor";

	private ExecutionContextKeys() {
	}
}

File: src/csharp/Foundgine.Core/Execution/ExecutionContextKeys.cs
namespace Foundgine.Core.Execution;

/// <summary>Reserved runtime context keys used by provider execution for dynamic request values.</summary>
public static class ExecutionContextKeys
{
    public const string PaginationLimit = "foundgine.pagination.limit";
    public const string PaginationOffset = "foundgine.pagination.offset";
    public const string PaginationHasCursor = "foundgine.pagination.hasCursor";
}
